package.json

```diff
@@ -62,7 +60,6 @@
     "mocha": "^5.1.1",
     "mocha-loader": "^1.1.3",
     "node-watch": "^0.5.8",
-    "prettier": "1.12.1",
```

Comment:
Does eslint do automatic file formatting? In other projects I've used both prettier and eslint together without issues

Reply:
@nick Yeah eslint does file formatting. Eslint does everything we need it to do - I don't see any use for prettier.

Reply:
Apparently prettier is [more powerful](https://github.com/prettier/prettier-eslint/issues/101) than eslint at formatting files. Perhaps we can run both eslint and prettier separately on demo-dapp, see which does a better job and decide to dump prettier based on that?